packages/calypso-build/jest/transform/README.md

```diff
@@ -7,7 +7,7 @@ i.e. they're not plain JavaScript:
 
 ```jsx
 const MyFlag = ( { countryCode } ) => (
-	<img alt="" src={ require( `flags/${ countryCode }.svg` ) } />
+	<img alt="" src={ import( `flags/${ countryCode }.svg` ) } />
```

Comment:
It's just a `README`, but this sets the `src` attribute to a promise of URL string. Can you really do this?

Reply:
nope, fixed